scripts/validate-file-path.js

```diff
@@ -0,0 +1,24 @@
+import { getFilePaths } from './utils.js';
```

Comment:
FYI it doesn't have to be in this PR but this file would be a good place to solve this: https://github.com/hyperlane-xyz/hyperlane-registry/issues/209

Reply:
Sounds good, we can discuss this later, but I was maybe thinking having a different file for checks related to chain and checks related to deployments